### DISCUSSION TITLE
Last-updated filter not working for Github projects

### DISCUSSION THREAD

**Discussion by jhuys** [community/community#108039] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

_Steps:_
1. Go to the default view of your project (`github.com/orgs/(organisation)/projects/(id)/views/1`)
2. Click on the filter input bar
3. Add a `last-updated`-based filter: for instance `last-updated:28days`

_Expected:_
Any issues that haven't been updated for the last 28 days are filtered out.

_Actual:_
No filtering takes place. All issues are shown instead.

_Update:_
It looks like the filtering stops working when:
- using a "last updated" period that is greater than 17 days
- omitting the `-` in front of `last-updated:17days`

So, it only works when `-last-updated:17days` or something similar is filled in.

_Links:_
This piece of documentation outlines how the `last-updated` filter should be used:
https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-for-when-an-item-was-last-updated

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by zsloane** [comment]
There seems to be a similar issue with the ```updated``` filter. Anything beyond about 2 weeks seems to not work.

I'm currently using ```updated:<@today-2w``` but as soon as I go farther back than that nothing is shown (even though there are plenty of issues that should be returned

**Comment by GeoJunkie** [comment]
There seems to be something that happened on January 27th. I check this periodically and the place where it breaks shifts. Today, it works with `last-updated:31days`, but not `last-updated:32days`, and it seems to consistently increment daily.
![CleanShot 2024-02-26 at 12 12 58@2x](https://github.com/community/community/assets/7596032/5b8624d3-7b5e-4112-8824-46271accd05a)

![CleanShot 2024-02-26 at 12 13 44@2x](https://github.com/community/community/assets/7596032/dfd41937-d759-43d5-8121-90eb803f4413)

**Comment by rileybroughten** [comment]
👋 Following a rollout on January 25th, 2024, there was an unexpected impact to the updated timestamps of items. If an `updated` filter is included specifying a date _after_ this time period, results should be returned as expected. We are working on a fix to resolve this issue for items updated _before_ this date, which is causing the unexpected results as pointed out above.

I'll post an update here once this is resolved. Apologies for the inconvenience and we appreciate your patience as we get this sorted out!

**Reply by tboddyspargo** [reply]
Any update, @rileybroughten ? I'm trying to filter an insights chart and noticing this limitation that doesn't align with documentation on project filters: 

![Screenshot 2025-02-19 at 17 17 22](https://github.com/user-attachments/assets/41ef95e1-e8ee-4dc8-88cf-58030a83a51e)

**Comment by jonmpoole80** [comment]
Has this been resolved because I am having this issue myself currently, however no variation of updated is working for me at all. If I remove it then I get results, as soon as I add it always 0 when it should be more than that.

**Comment by neilzilla** [comment]
Any update on this? it's 2026 and I'm experiencing the same thing
